// _vuepress@0.14.11@vuepress/lib/app/.temp/routes.js
import Vue from 'vue'
Vue.component("about", () => import("/Users/xuchangjian/workspace/my/pc/docs/.vuepress/components/about.vue"))
import ThemeLayout from '@themeLayout'
import ThemeNotFound from '@themeNotFound'
import { injectMixins } from '@app/util'
import rootMixins from '@app/root-mixins'

injectMixins(ThemeLayout, rootMixins)
injectMixins(ThemeNotFound, rootMixins)

export const routes = [
  {
    name: "v-913b8085677fe",
    path: "/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/README.md").then(comp => {
        Vue.component("v-913b8085677fe", comp.default)
        next()
      })
    }
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-fe3131da397ef",
    path: "/src/html/about.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/html/about.md").then(comp => {
        Vue.component("v-fe3131da397ef", comp.default)
        next()
      })
    }
  },
  {
    name: "v-f7229930503fa",
    path: "/src/rules/rules.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/rules/rules.md").then(comp => {
        Vue.component("v-f7229930503fa", comp.default)
        next()
      })
    }
  },
  {
    name: "v-2fa9484d75c82",
    path: "/src/web/node/express.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/web/node/express.md").then(comp => {
        Vue.component("v-2fa9484d75c82", comp.default)
        next()
      })
    }
  },
  {
    name: "v-4492633833bc1",
    path: "/src/web/typescript/ts-base-types.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/web/typescript/ts-base-types.md").then(comp => {
        Vue.component("v-4492633833bc1", comp.default)
        next()
      })
    }
  },
  {
    name: "v-15ec96fdaf1",
    path: "/src/web/typescript/ts-interface-types.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/web/typescript/ts-interface-types.md").then(comp => {
        Vue.component("v-15ec96fdaf1", comp.default)
        next()
      })
    }
  },
  {
    name: "v-77f6464d11366",
    path: "/src/web/typescript/ts-introduce.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/web/typescript/ts-introduce.md").then(comp => {
        Vue.component("v-77f6464d11366", comp.default)
        next()
      })
    }
  },
  {
    name: "v-4bd3d2539267e",
    path: "/src/web/vue/vuex.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/web/vue/vuex.md").then(comp => {
        Vue.component("v-4bd3d2539267e", comp.default)
        next()
      })
    }
  },
  {
    path: '*',
    component: ThemeNotFound
  }
]
