// _vuepress@0.14.11@vuepress/lib/app/.temp/routes.js
import Vue from 'vue'
Vue.component("about", () => import("/Users/xuchangjian/workspace/my/pc/docs/.vuepress/components/about.vue"))
import ThemeLayout from '@themeLayout'
import ThemeNotFound from '@themeNotFound'
import { injectMixins } from '@app/util'
import rootMixins from '@app/root-mixins'

injectMixins(ThemeLayout, rootMixins)
injectMixins(ThemeNotFound, rootMixins)

export const routes = [
  {
    name: "v-ab67793735a99",
    path: "/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/README.md").then(comp => {
        Vue.component("v-ab67793735a99", comp.default)
        next()
      })
    }
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-2841d51b20e83",
    path: "/src/html/about.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/html/about.md").then(comp => {
        Vue.component("v-2841d51b20e83", comp.default)
        next()
      })
    }
  },
  {
    name: "v-ce23da9a0a6e5",
    path: "/src/rules/rules.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/rules/rules.md").then(comp => {
        Vue.component("v-ce23da9a0a6e5", comp.default)
        next()
      })
    }
  },
  {
    name: "v-e5d864077431",
    path: "/src/web/node/express.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/web/node/express.md").then(comp => {
        Vue.component("v-e5d864077431", comp.default)
        next()
      })
    }
  },
  {
    name: "v-29179e05d86ab",
    path: "/src/web/vue/vuex.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/web/vue/vuex.md").then(comp => {
        Vue.component("v-29179e05d86ab", comp.default)
        next()
      })
    }
  },
  {
    path: '*',
    component: ThemeNotFound
  }
]
