// _vuepress@0.14.11@vuepress/lib/app/.temp/routes.js
import Vue from 'vue'

import ThemeLayout from '@themeLayout'
import ThemeNotFound from '@themeNotFound'
import { injectMixins } from '@app/util'
import rootMixins from '@app/root-mixins'

injectMixins(ThemeLayout, rootMixins)
injectMixins(ThemeNotFound, rootMixins)

export const routes = [
  {
    name: "v-cbcaff0cb6b79",
    path: "/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/README.md").then(comp => {
        Vue.component("v-cbcaff0cb6b79", comp.default)
        next()
      })
    }
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-d70a849b11327",
    path: "/src/web/express.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/web/express.md").then(comp => {
        Vue.component("v-d70a849b11327", comp.default)
        next()
      })
    }
  },
  {
    name: "v-868e57349c45e",
    path: "/src/web/vuex.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/xuchangjian/workspace/my/pc/docs/src/web/vuex.md").then(comp => {
        Vue.component("v-868e57349c45e", comp.default)
        next()
      })
    }
  },
  {
    path: '*',
    component: ThemeNotFound
  }
]
